File: app/vn/vnext/sefuri/sf/json/SF00302/request/SF0030203Req.java
package vn.vnext.sefuri.sf.json.SF00302.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import vn.vnext.sefuri.sf.json.SF00302.model.PaperModalJson;
import vn.vnext.sefuri.sf.json.SF00302.model.ProductJson;
import vn.vnext.sefuri.sf.json.common.AbstractJson;

import java.util.List;

/**
 * Created by VuPT on 2/27/2017.
 */
public class SF0030203Req extends AbstractJson {
    @JsonProperty("product")
    private ProductJson productJson;

    public ProductJson getProductJson() {
        return productJson;
    }

    public void setProductJson(ProductJson productJson) {
        this.productJson = productJson;
    }

}

File: client/src/app/request/SF0030203.req.ts
import {Product} from "../model/core/Product.model";
import {PaperModel} from "../view/SF/SF00302/model/paper.model";

export class SF0030203Req {
    product: Product;
    paperNews: PaperModel[];

    constructor( product: Product, paperNews?: PaperModel[]) {
        this.product = product;
        this.paperNews = paperNews;
    }
}
